public/jira/rolledup/work-type/work-type.test.js

```diff
@@ -1,132 +1,233 @@
 // sum.test.js
-import { expect, test } from 'vitest'
-import { rollupDatesByWorkType, rollupWorkTypeDates } from './work-type.js'
-import { addReportingHierarchy, addChildrenFromGroupedHierarchy } from '../../rollup/rollup.js';
-
-
-function maintainOrderByKey(source, updated){
-    const keyToIndex = {};
-    for(let i = 0; i < source.length; i++) {
-        keyToIndex[source[i].key] = i;
+import { expect, test } from "vitest";
+import { rollupDatesByWorkType, rollupWorkTypeDates } from "./work-type.js";
```

Comment:
Should this fine be ts?

Reply:
No this file `rolledup/work-type/work-type.js` is still a js file. Still to be migrated to typescript